feature/account/edit/src/main/kotlin/app/k9mail/feature/account/edit/ui/server/settings/save/SaveServerSettingsScreen.kt

```diff
@@ -0,0 +1,100 @@
+package app.k9mail.feature.account.edit.ui.server.settings.save
+
+import androidx.activity.compose.BackHandler
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import app.k9mail.core.ui.compose.common.mvi.observe
+import app.k9mail.core.ui.compose.designsystem.template.Scaffold
+import app.k9mail.core.ui.compose.theme.K9Theme
+import app.k9mail.core.ui.compose.theme.ThunderbirdTheme
+import app.k9mail.feature.account.common.ui.AccountTopAppBarWithBackButton
+import app.k9mail.feature.account.common.ui.WizardNavigationBar
+import app.k9mail.feature.account.common.ui.WizardNavigationBarState
+import app.k9mail.feature.account.edit.ui.server.settings.save.SaveServerSettingsContract.Effect
+import app.k9mail.feature.account.edit.ui.server.settings.save.SaveServerSettingsContract.Event
+import app.k9mail.feature.account.edit.ui.server.settings.save.SaveServerSettingsContract.ViewModel
+import app.k9mail.feature.account.edit.ui.server.settings.save.fake.FakeSaveServerSettingsViewModel
+
+@Composable
+fun SaveServerSettingsScreen(
+    onNext: () -> Unit,
+    onBack: () -> Unit,
+    viewModel: ViewModel,
+    modifier: Modifier = Modifier,
+) {
+    val (state, dispatch) = viewModel.observe { effect ->
+        when (effect) {
+            is Effect.NavigateNext -> onNext()
+            Effect.NavigateBack -> onBack()
+        }
+    }
+
+    LaunchedEffect(key1 = Unit) {
+        dispatch(Event.SaveServerSettings)
+    }
```

Comment:
This will trigger an additional save operation e.g. after a configuration change. We have the same problem in a couple of other places. I'll create an issue for that.